colonialsite/colonialsite/settings.py

```diff
@@ -15,59 +15,109 @@
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
-# Path to file uploads (upload images)
-MEDIA_ROOT = os.path.join(BASE_DIR, 'media')
-MEDIA_URL = '/media/'
-
-# Quick-start development settings - unsuitable for production
-# See https://docs.djangoproject.com/en/1.9/howto/deployment/checklist/
 
-# SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = '-j_7bebp&w9&()_nn+*dmg93%5qxy#$8tz)()2e-d&=ne#767p'
+# --- IMPORTANT SETTINGS ---
 
-# SECURITY WARNING: don't run with debug turned on in production!
+# SECRET_KEY
 DEBUG = True
 
-ALLOWED_HOSTS = []
+# EMAIL_HOST_USER
+# EMAIL_HOST_PASSWORD
 
-# --- NEW SETTINGS ---
-
-SOPHOMORE_YEAR = '2019'
-JUNIOR_YEAR = '2018'
-SENIOR_YEAR = '2017'
+# TIGERBOOK_LOGIN
+# TIGERBOOK_PW
 
 # Used in coloauth.middleware
-# Default Login is automatically added
+# Default Login is automatically added 
```

Comment:
Did you add this trailing whitespace back in

Reply:
Sorry, I copypasta'd from an old version.